lib/l10n/app_en.arb

```
@@ -501,6 +501,47 @@
+  "defaultLanguage": "Default Language",
+  "selectLanguage": "Select Language",
+  "noDefaultLanguage": "No default language",
+  "noPromptsAvailable": "No prompts available",
+  "createPromptsFirst": "Create AI prompts first to configure them here",
+  "selectAllowedPrompts": "Select which prompts are allowed for this category",
+  "audioRecordings": "Audio Recordings",
+  "images": "Images",
+  "taskSummaries": "Task Summaries",
+  "noPromptsForType": "No prompts available for this type",
+  "errorLoadingPrompts": "Error loading prompts",
+  "categoryDeleteQuestion": "Delete Category?",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)